core/standalone/README.md

```diff
@@ -93,23 +93,28 @@ $ java -jar openwhisk-standalone.jar -h
   -d, --data-dir  <arg>            Directory used for storage
       --dev-kcf                    Enables KubernetesContainerFactory for local
                                    development
-      --dev-mode                   Developer mode speeds up the startup by
-                                   disabling preflight checks and avoiding
-                                   explicit pulls.
-      --disable-color-logging      Disables colored logging
-      --kafka-docker-port  <arg>   Kafka port for use by docker based services.
-                                   If not specified then 9091 or some random
-                                   free port (if 9091 is busy) would be used
-      --kafka-port  <arg>          Kafka port. If not specified then 9092 or
-                                   some random free port (if 9092 is busy) would
-                                   be used
-  -p, --port  <arg>                Server port
+      --dev-mode                      Developer mode speeds up the startup by
+                                      disabling preflight checks and avoiding
+                                      explicit pulls.
+      --dev-user-events-port  <arg>   Specify the port for the user-event
```

Comment:
Would it make sense to have <arg> optional and default to 9095? If user-events is run with an IDE, it will start on this port by default.

Reply:
Tried to support that but [scallop](https://github.com/scallop/scallop) arg parser does not support it. If I specify `--dev-user-events-port` then need to provide an explicit value otherwise it gives an error like below

```
[openwhisk] Error: Bad arguments for option 'dev-user-events-port': '' - you should provide exactly one argument for this option
```